lib/hexpm/web/views/package_view.ex

```diff
@@ -114,12 +114,27 @@ defmodule Hexpm.Web.PackageView do
   end
 
   def retirement_message(retirement) do
-    [ReleaseRetirement.reason_text(retirement.reason)] ++
-      if(retirement.message, do: [": ", retirement.message], else: [])
+    ["Retired package"] ++
+      cond do
+        ReleaseRetirement.reason_text(retirement.reason) && retirement.message ->
```

Comment:
Bind `ReleaseRetirement.reason_text/1` result in a variable since we call it so many times.